app/models/katello/sync_plan.rb

```diff
@@ -55,6 +55,10 @@ def plan_date_time
       self.sync_date.strftime('%Y/%m/%d %H:%M:%S %Z')
     end
 
+    def sync_date_utc
+      self.sync_date.strftime('%Y/%m/%d %H:%M:%S %z')
```

Comment:
This actually appears to work in all cases that I tried so we may not even need the conditional at all, we could just use `%z` instead of `%Z`.

From [the docs](https://ruby-doc.org/core-2.2.0/Time.html#method-i-strftime):
```
Time zone:
  %z - Time zone as hour and minute offset from UTC (e.g. +0900)
          %:z - hour and minute offset from UTC with a colon (e.g. +09:00)
          %::z - hour, minute and second offset from UTC (e.g. +09:00:00)
  %Z - Abbreviated time zone name or similar information.  (OS dependent)
```

Note the OS dependent for `%Z`.  It seems like that is a value that was meant for display only and not date/time math and conversions.